packages/@aws-cdk/aws-apigateway/lib/stage.ts

```diff
@@ -348,4 +350,93 @@ export class Stage extends Resource implements IStage {
       };
     }
   }
+
+  /**
+   * Returns the given named metric for this stage
+   */
+  public metric(metricName: string, props?: cloudwatch.MetricOptions): cloudwatch.Metric {
```

Comment:
similar comment as above.

Additionally, having `metric()` next to `metricClientError()` is confusing; can we add a more descriptive name to `metric()`, to better distinguish it from the other `metric*()` methods in both classes?

Reply:
Again, the naming `metric()` here seems to match the naming in other modules such as SQS and Lambda, maybe it is better to leave them consistent?